cmake/SelectPcsx2Plugins.cmake

```diff
@@ -89,26 +88,6 @@ endif()
 # be build.
 #-------------------------------------------------------------------------------
 
-#---------------------------------------
-#			CDVDnull
-#---------------------------------------
-if(GTKn_FOUND)
-    set(CDVDnull TRUE)
-endif()
-#---------------------------------------
-
-#---------------------------------------
-#			cdvdGigaherz
-#---------------------------------------
-if(NOT EXISTS "${CMAKE_SOURCE_DIR}/plugins/cdvdGigaherz" OR NOT Linux)
-    set(cdvdGigaherz FALSE)
-elseif(Linux AND GTKn_FOUND AND LIBUDEV_FOUND)
```

Comment:
The pcsx2 core will have a libudev dependency on Linux if cdvdGigaherz code is moved into it. I do not see this reflected in this cmake file.

Reply:
I already included that dependency. It's in CMakeLists.txt line 650